src/main/java/com/example/ShortnerURL/Controller/PingController.java

```diff
@@ -15,16 +17,18 @@ public class PingController {
     private JdbcTemplate jdbcTemplate;
 
     @GetMapping("/ping")
-    public ResponseEntity<Map<String, String>> ping() {
+    public ResponseEntity<Object> ping() {
         try {
             jdbcTemplate.queryForObject("SELECT current_timestamp;", String.class);
-            Map<String, String> response = new HashMap<>();
-            response.put("status", "ok");
-            return new ResponseEntity<>(response, HttpStatus.OK);
+            SuccessResponseDto successResponseDto = SuccessResponseDto.builder()
+                    .status("status: ok")
```

Comment:
can you check the exact body that is being returned when you run this, and see if it is matching the expected body.

Reply:
I changed the status line and now return the correct body.